init_project.yml

```diff
@@ -24,3 +24,7 @@
       when: item is not none
 
     - import_tasks: tasks/create_route_aliases.yml
+      tags: route
+
+    - import_tasks: tasks/create_fixed_routes.yml
```

Comment:
What about static routes instead of fixed routes?

Reply:
Yes